exporter/sapmexporter/go.mod

```diff
@@ -4,7 +4,6 @@ go 1.12
 
 require (
 	github.com/open-telemetry/opentelemetry-collector v0.2.1-0.20191119152140-567e1046cefa
-	github.com/pierrec/lz4 v2.0.5+incompatible // indirect
```

Comment:
Every time that I open repo using Goland it adds this back, I have to check my settings...